[DOCS-12582] Agentic Onboarding RUM (#32743)
DataDog/documentation: content/en/real_user_monitoring/application_monitoring/react_native/setup/reactnative.md

## Setup

> Follow the step-by-step instructions to manually add and configure the RUM SDK in your React Native application.

Reviewer: ```suggestion
Follow these instructions to manually add and configure the RUM SDK in your React Native application.
```

## Setup

**Choose your setup method:**

> - **[Agentic Onboarding (in Preview)][15]**: Use AI coding agents (Cursor, Claude Code) to automatically instrument your React Native application with one prompt. The agent detects your project structure and configures the RUM SDK for you.

Reviewer 2: Unfortunately, Agentic Onboarding doesn't support React Native yet, so we probably should exclude this here

[12]: https://support.apple.com/guide/security/security-of-runtime-process-sec15bfe098e/web
[13]: https://stackoverflow.com/questions/37388126/use-frameworks-for-only-some-pods-or-swift-pods/60914505#60914505
[14]: https://reactnative.dev/architecture/landing-page

> [15]: /real_user_monitoring/application_monitoring/ios/agentic_onboarding/?tab=realusermonitoring

Reviewer: This isn't used anywhere. Did you mean to update the Setup section to include the "Agentic onboading vs manual setup" section you added to the other pages?
